pages/common/npx.md

```diff
@@ -0,0 +1,15 @@
+# npx
+
+> Execute binaries from npm packages.
+
+- Execute the command:
+
+`npx <command>[@version] [command-arg]`
+
+- Define the package to be installed, then execute the binaries:
+
+`npx [options] [-p|--package <pkg>]... <command> [command-arg]...`
+
+- Call help see other options meaning with local languange:
```

Comment:
This seems to be grammatically incorrect. And there is no need to add "local language". Perhaps "View help contents" is good enough.